### PR TITLE
Arnej/add java tensor conformance app

### PR DESCRIPTION
**Pull request by arnej27959**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@lesters please review
@havardpe please review
@bratseth @geirst FYI

### CODE REVIEW

File: searchlib/src/main/java/com/yahoo/searchlib/tensor/EvaluateTensorConformance.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (lesters) [inline searchlib/src/main/java/com/yahoo/searchlib/tensor/EvaluateTensorConformance.java:1]: Copyright Verizon Media

File: searchlib/src/main/java/com/yahoo/searchlib/tensor/EvaluateTensorConformance.java [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        } catch (IOException e) {
+            System.err.println("FAILED writing output: "+e);
+            System.exit(1);
+        }
+    }
+        
+    private boolean testCase(String test, int count) {
+        boolean okAndEqual = false;
+        try {
+	    Slime input = SlimeUtils.jsonToSlime(test);
```
Comment (lesters) [inline searchlib/src/main/java/com/yahoo/searchlib/tensor/EvaluateTensorConformance.java:77]: indent

**Review by lesters — APPROVED** [review]
LGTM!

**Review by havardpe — COMMENTED** [review]
I suggest removing the comparison with expected result, since this will be done by the cpp binary afterwords anyway.

**Pull request merged**